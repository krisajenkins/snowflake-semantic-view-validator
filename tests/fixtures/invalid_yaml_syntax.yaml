name: INVALID
description: This has invalid YAML syntax
tables:
  - name: TEST
    base_table:
      database: DB
      schema: PUBLIC
      table: TEST
    dimensions:
      - name: ID
        expr: ID
        data_type: VARCHAR
      - name: MISSING_COLON
        expr ID_EXPR
        data_type: VARCHAR
