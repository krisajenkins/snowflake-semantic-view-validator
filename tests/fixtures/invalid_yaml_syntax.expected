════════════════════════════════════════════════════════════════════════════════
  VALIDATION ERROR
════════════════════════════════════════════════════════════════════════════════

✗ Failed to parse YAML file: could not find expected ':' at line 15 column 9, while scanning a simple key at line 14 column 9

TIP:
  Check the YAML syntax at the indicated line and column.
  Common issues include:
    • Incorrect indentation (use spaces, not tabs)
    • Missing colons after keys
    • Unquoted strings containing special characters
    • Missing required fields

════════════════════════════════════════════════════════════════════════════════